README.md

```diff
@@ -1,2 +1,89 @@
-## Loopster
-Сервисная библиотека, которая обеспечивает выполнение сервисных процессов, следит за их зависаниями и при необходимости осуществляет перезапуск.
+# Loopster
+
+Loopster is a service library that provides mechanisms for executing and managing services, including monitoring their status and automatically restarting them when necessary.
+
+## Installation
+
+To install Loopster, you can use pip:
+
+```
+pip install loopster
+```
+
+## Usage
+
+### Basic Service Example
+
+Here's an example of how to create and run a basic service using Loopster:
+
+```python
+from loopster.hubs import base
+from loopster.hubs.controllers import panic
+from loopster.hubs.drivers import process
+from loopster.services import softirq
+
+class BasicService(softirq.SoftIrqService):
+    def _step(self):
+        print("Hello, world!")
+
+driver = process.ProcessDriver()
+controller = panic.PanicController()
+hub = base.BaseHub(driver=driver, controller=controller)
+
+hub.add_service(svc_class=BasicService, svc_kwargs={})
+hub.serve()
+```
+
+### Nested Service Example with MySQLWrapper
+
+Here's an example of how to use a nested service with the `mysqlwrapper` module:
+
+```python
+from restalchemy.storage.sql import engines
+from rooster.hubs.controllers import force_state
+from loopster.hubs.drivers import process
+from loopster.services import mysql
+from loopster.services import softirq
+
+class BasicService(softirq.SoftIrqService):
+    def _step(self):
+        print("Hello, world!")
+
+hub = process.ProcessHub(
+    controller=force_state.AlwaysForceTargetStateController())
+
+hub.add_service(
+    svc_class=mysql.MySQLStorageWrapper,
+    svc_kwargs=dict(
+        nested_class=BasicService,
+        nested_kwargs=dict(
+            step_period=STEP_PERIOD,
+        ),
+        mysql_connection_url=engines.DBConnectionUrl(CONNECTION_URL)
+    )
+)
+```
+
+## Public Interface
+
+### Services
+
+Loopster provides several built-in services, including:
```

Comment:
How about EtcdSingleSoftIrqService?

Reply:
```
class EtcdSingleSoftIrqService(softirq.SoftIrqService):
    """First attempt at etcd-backed locking implementation

    This class is deprecated, all of it's logic and more is implemented
    in SoftIrqService
```
    
It's deprecated, we may even want to remove it in future